.golangci.yaml

```diff
@@ -48,16 +47,13 @@ linters:
     - nakedret
     - noctx
     - nolintlint
-    - rowserrcheck
-    - scopelint
+    - revive
     - staticcheck
-    - structcheck
```

Comment:
Anything removed here is because it is deprecated or removed in the latest versions. Additions were recommended by the tool as replacements.